src/effects/CompressionMeterPanel.cpp

```diff
@@ -43,19 +43,23 @@ EVT_TIMER(timerId, CompressionMeterPanel::OnTimer)
 END_EVENT_TABLE()
 
 CompressionMeterPanel::CompressionMeterPanel(
-   wxWindow* parent, CompressorInstance& instance)
-    : wxPanelWrapper { parent, wxID_ANY }
+   wxWindow* parent, int id, CompressorInstance& instance, float dbRange)
+    : wxPanelWrapper { parent, id }
     , mMeterValuesQueue { std::make_unique<
          DynamicRangeProcessorMeterValuesQueue>(audioFramesPerTick) }
-    , mPlaybackStartStopSubscription {
-       static_cast<InitializeProcessingSettingsPublisher&>(instance).Subscribe(
-          [&](const std::optional<InitializeProcessingSettings>& evt) {
-             if (evt)
-                Reset();
-             else
-                mStopWhenZero = true;
-          })
-    }
+    , mPlaybackStartStopSubscription { static_cast<
```

Comment:
nitpick: did the autoformatter do it? It was way easier to read before

Reply:
Yeah, this must be a spec of the `.clang-format` you can find in some of our subdirectories.